lib/pymedphys/_experimental/streamlit/apps/sum_doses.py

```diff
@@ -0,0 +1,318 @@
+"""Copyright (C) 2021 Matthew Jennings
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import copy
+import pathlib
+from typing import List, Sequence, Union
+
+from pymedphys._imports import numpy as np
+from pymedphys._imports import pydicom
+from pymedphys._imports import streamlit as st
+
+from pymedphys._dicom.coords import xyz_axes_from_dataset
+from pymedphys._dicom.dose import dose_from_dataset
+from pymedphys._streamlit import categories
+
+CATEGORY = categories.PRE_ALPHA
+TITLE = "Sum Coincident DICOM Doses"
+
+HERE = pathlib.Path(__file__).parent.resolve()
+
+
+def _check_files_valid(
+    files: Sequence[pathlib.Path],
+) -> Union["List[pydicom.dataset.Dataset]", None]:
```

Comment:
```suggestion
) -> Optional[List["pydicom.dataset.Dataset"]]:
```

See https://docs.python.org/3/library/typing.html#typing.Optional